src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```diff
@@ -62,6 +62,7 @@ Copyright (c) .NET Foundation. All rights reserved.
           This is to reduce 'tearing'/dependency mismatch, but as always users can override this behavior by disabling the hosted compiler flag.
           This is done in Common.targets so that it applies to ApiCompat which uses RoslynTargetsPath in the outer build. -->
     <BuildWithNetFrameworkHostedCompiler Condition="'$(BuildWithNetFrameworkHostedCompiler)' == ''
+                                                    and '$(RoslynUseSdkCompiler)' != 'true'
```

Comment:
We have a lot of properties that we use now (looking at the file above and this). Do we have a place where they are all written down that we can reference in the future?

Reply:
Probably not, but maybe we could do that in one of the docs `torn-sdk.md` or `decouple-vs-and-net-sdk.md`? I'm not sure why we have both docs actually, seems like the former one could be removed?

Reply:
We have this document which is intended for customers and lists the exposed sdk properties: https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props

Reply:
- https://github.com/dotnet/docs/pull/45620